roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -211,6 +203,9 @@
       EOF
     executable: /bin/bash
   when:
+    - global_env.OS_BRANCH != 'stable/mitaka'
```

Comment:
@liusheng Please help to review the PR, thanks

Reply:
I modified the name of the task, if don't match the conditions, the code won't execute, I think just the name of the task print, so I modified the task name. Is it ok?

Reply:
yes, see example:
```
2019-07-21 20:02:46.168005 | TASK [create-devstack-local-conf : create devstack local conf with Heat enabled]
2019-07-21 20:02:46.306743 | ubuntu-trusty | skipping: Conditional result was False
```

Reply:
OK, thank you.